DOCS/man/options.rst

```diff
@@ -5286,6 +5286,12 @@ them.
         ``--scale-param1`` and ``--scale-param2``. This filter is very good at
         downscaling (see ``--dscale``).
 
+    ``catmull_rom``
+        Catmull-Rom. A Cubic filter in the same vein as ``mitchell``, where
+        the ``B`` and ``C`` parameters are ``0.0`` and ``0.5`` respectively.
+        Nominally referred to as Bicubic. This filter is sharper than
```

Comment:
users might be confused since `--scale=help` already shows a filter called "bicubic"
not sure if it's useful to clear that up here

Reply:
I prefaced the sentence with "nominally" here to hopefully signal that even though it's called "Bicubic" at times, `catmull_rom` is just another Cubic filter variant among many others. I was mirroring the description for `ewa_lanczos`, where it also says:

https://github.com/mpv-player/mpv/blob/6f83529f219ab5ac546013967419c00e6ff3edfc/DOCS/man/options.rst?plain=1#L5270-L5271

...even though `--scale=jinc` is already a mapped option. So, the line here is referencing a colloquial alias for `ewa_lanczos`, instead of the actual `jinc` function.

Reply:
On second thought, I think you're right. A lot of applications use various cubic splines other than `catmull_rom` and just call them "Bicubic" regardless. I might've been too hyper-focused on GIMP and VapourSynth for no reason. Meanwhile, the "Jinc" alias is really only used by one other application that I know of, and it only refers to `ewa_lanczos`. The libplacebo README also makes a reference to "Jinc", and I've seen the plplay application refer to ewa_lanczos as Jinc.

So, the degrees of ambiguity here between Jinc and Bicubic aren't really comparable, Ideally, there should be _no_ ambiguity at all, but for the sake of brevity I'll just delete the line.